hdf5-vol-daos.spec

```diff
@@ -175,6 +176,7 @@ HDF5 VOL DAOS tests with mpich
 %setup -n vol-daos-%{vol_tag}
 %setup -T -D -b 1 -n vol-daos-%{vol_tag}
 %patch0 -p1 -b .3574df56fed78ee70172f17a528fae7d90051d58.patch
+%patch1 -p1 -b .b64776874c07732720b716b81b8dc9c14115da12.patch
 
```

Comment:
I don't want to give you any rpm headache so feel free to ignore but I think in theory you should be able to combine all the 4 lines above into `%autosetup -p1 -n vol-daos-%{vol_tag}` ? then it should apply all patches by itself. Or if there is any need to keep the first 2 lines you could also do `%autopatch -p1` instead of manually specifying each patch.

Reply:
yea sorry im not familiar with all this. just doing it like it was done before when we needed to apply patches over a release. we can do improvements in a separate PR ;-)

Reply:
I'm not sure that `%autosetup` would handle expanding the multiple `Source` tarballs in the same way that the multiple *%setup*s are here.  Wouldn't hurt to try on the next go around with this repo.